main_cmd.py

```diff
@@ -0,0 +1,45 @@
+"""HummingBird execution file.
+
+This is a python excution file to run I2C eletrical test on
+data from csv file.
+
+"""
+import argparse
+import os
+import subprocess
+import time
+
+from hummingbird_cmd import HummingBird
+
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser()
+  parser.add_argument("csv", help="csv data path")
+  parser.add_argument("--output_folder", default=None,
+                      help="output report folder")
+  parser.add_argument("--working_voltage", default=None, type=float,
```

Comment:
You might want to add [choices=list-of-allowed-values](https://docs.python.org/3/library/argparse.html#choices) here and for operation_mode